doc/lua_api.md

```diff
@@ -2763,15 +2764,19 @@ Elements
 * End of a container, following elements are no longer relative to this
   container.
 
-### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<orientation>;<scroll factor>]`
+### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<orientation>;<content padding>]`
 
 * Start of a scroll_container block. All contained elements will ...
   * take the scroll_container coordinate as position origin,
   * be additionally moved by the current value of the scrollbar with the name
     `scrollbar name` times `scroll factor` along the orientation `orientation` and
   * be clipped to the rectangle defined by `X`, `Y`, `W` and `H`.
 * `orientation`: possible values are `vertical` and `horizontal`.
-* `scroll factor`: optional, defaults to `0.1`.
```

Comment:
broken backwards compatibility

Reply:
Apparently I did not only remove the `"auto"` value that I previously implemented but the entire parameter.

Fixed in c743ed5d8.